core/templates/services/html-length.service.ts

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//      http://www.apache.org/licenses/LICENSE-2.0
+// http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Ditto, please don't change parts that you don't need to change. Make sure to check your changes using the "Files Changed" tab before requesting a review.